content/v2.0/influxdb-templates/use.md

```diff
@@ -1,20 +1,193 @@
-  - Define secrets while installing a template
+Use the `influx pkg` command to summarize, validate, and install templates from
+your local filesystem and from URLs.
+
+- [View a template summary](#view-a-template-summary)
+- [Validate a template](#validate-a-template)
+- [Install templates](#install-templates)
+- [Use InfluxDB community templates](#use-influxdb-community-templates)
+
+## View a template summary
+To view a summary of what's included in a template before installing the template,
+use the [`influx pkg summary` command](/v2.0/reference/cli/influx/pkg/summary/).
+Summarize templates stored in your local filesystem or from a URL.
+
+{{% code-tabs-wrapper %}}
+{{% code-tabs %}}
+[From a file](#)
+[From a URL](#)
+{{% /code-tabs %}}
+{{% code-tab-content %}}
+```sh
+# Syntax
+influx pkg summary -f <template-file-path>
+
+# Example
+influx pkg summary -f /path/to/template.yml
+```
+{{% /code-tab-content %}}
+{{% code-tab-content %}}
+```sh
+# Syntax
+influx pkg summary -u <template-url>
+
+# Example
+influx pkg summary -u https://raw.githubusercontent.com/influxdata/community-templates/master/linux_system/linux_system.yml
+```
+{{% /code-tab-content %}}
+{{% /code-tabs-wrapper %}}
+
+## Validate a template
+To validate a template before installing it or for troubleshooting purposes, use
+the [`influx pkg validate` command](/v2.0/reference/cli/influx/pkg/validate/).
+Validate templates stored in your local filesystem or from a URL.
+
+{{% code-tabs-wrapper %}}
+{{% code-tabs %}}
+[From a file](#)
+[From a URL](#)
+{{% /code-tabs %}}
+{{% code-tab-content %}}
+```sh
+# Syntax
+influx pkg validate -f <template-file-path>
+
+# Example
+influx pkg validate -f /path/to/template.yml
+```
+{{% /code-tab-content %}}
+{{% code-tab-content %}}
+```sh
+# Syntax
+influx pkg validate -u <template-url>
+
+# Example
+influx pkg validate -u https://raw.githubusercontent.com/influxdata/community-templates/master/linux_system/linux_system.yml
+```
+{{% /code-tab-content %}}
+{{% /code-tabs-wrapper %}}
+
+## Install templates
+Use the [`influx pkg` command](/v2.0/reference/cli/influx/pkg/) to install templates
+from your local filesystem or from URLs.
+
+- [Install templates from files](#install-templates-from-files)
+- [Install templates from URLs](#install-templates-from-urls)
+- [Install templates from both files and URLs](#install-templates-from-both-files-and-urls)
+- [Include secrets when installing a template](#include-secrets-when-installing-a-template)
+
+### Install templates from files
+To install templates stored on your local machine, use the `-f` or `--file` flag
+to provide the **file path** of the template manifest.
+
+```sh
+# Syntax
+influx pkg -f <template-file-path>
+
+# Examples
+# Install a single template
+influx pkg -f /path/to/template.yml
+
+# Install multiple templates
+influx pkg \
+  -f /path/to/this/template.yml \
+  -f /path/to/that/template.yml
+```
+
+#### Install all templates in a directory
```

Comment:
maybe break out how to install multiple templates in a directory into it's own section?